src/renderer/components/ft-share-button/ft-share-button.scss

```
@@ -56,6 +57,7 @@
 
       .pastelPink &,
       .light &,
+      .gruvboxLight &,
```

```suggestion
      .light &,
```

Remove extraneous reference to gruvBoxLight